perf/terraform/terraform.tf

```diff
@@ -2,37 +2,78 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "4.67.0"
     }
   }
 }
 
-module "server_region" {
-  source = "./region"
-  region = "us-west-2"
-  ami = "ami-0747e613a2a1ff483"
-
-  common_tags = {
+locals {
+  tags = {
     Project = "perf"
   }
+  root = "."
 }
 
-module "client_region" {
-  source = "./region"
-  region = "us-east-1"
-  ami = "ami-06e46074ae430fba6"
+variable "common_enabled" {
+  type        = bool
+  description = "Whether or not to create common resources"
+  default     = false
+}
```

Comment:
> Testing against a personal AWS account doesn't require the AWS Lambda.

Using this flag we can now control whether the lambda is created or not. By default it's disabled. So when running this locally, it won't create any lambdas in a personal account.

Reply:
Very neat! Thank you.